src/panels/config/integrations/integration-panels/ssdp/dialog-ssdp-discovery-info.ts

```diff
@@ -39,6 +40,16 @@ class DialogSSDPDiscoveryInfo extends LitElement implements HassDialog {
     });
   }
 
+  private _showRawData(key: string, data: unknown) {
+    return (e: Event) => {
+      e.preventDefault();
+      showSSDPRawDataDialog(this, {
+        key,
+        data: data as Record<string, unknown>,
```

Comment:
```suggestion
  private _showRawData(key: string, data: Record<string, unknown>) {
    return (e: Event) => {
      e.preventDefault();
      showSSDPRawDataDialog(this, {
        key,
        data,
```
Better to cast it in the call below where you already do some type checking